test/integration/languages_int_test.go

```diff
@@ -57,13 +57,8 @@ func (suite *LanguagesIntegrationTestSuite) TestLanguages_install() {
 	cp.Expect("python")
 	cp.ExpectExitCode(0)
 
-	cp = ts.Spawn("languages", "install", "python")
-	cp.Expect("Language: python is already installed")
-	cp.ExpectExitCode(1)
-	ts.IgnoreLogErrors()
```

Comment:
Isn't this still a valid test?

Reply:
I didn't think so because we are now fully utilizing the buildplanner, and it'll tell us if there's nothing new in the commit (i.e. the package/language is already installed), rather than us manually fetching the project's current language and doing a comparison in the runner.

Reply:
Ahh okay, that makes sense. I still think we could have an expect here just to ensure that this command fails sensibly, but I'll leave that up to you.

Reply:
In this case, it will update Python because the checkout version is old. You actually have to run the install command twice before it will fail with "already" installed, and I'm concerned that it'll just balloon the integration test run time. Also, in my testing of this, macOS tended to have more build failures with Python 3.11, so I'd rather just defer to the existing `TestPackage_Duplicate()` to raise an error like this.